### PR TITLE
Create ui-c8y-1020-21-0-firmware-list-to-grid-view-migration.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3746](https://cumulocity.atlassian.net/browse/DM-3746)
Original PR: [6971](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6971)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-21-0-firmware-list-to-grid-view-migration.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Firmware list to grid view migration.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-21-0-firmware-list-to-grid-view-migration.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Firmware list migrated to grid view
```

File: content/change-logs/device-management/ui-c8y-1020-21-0-firmware-list-to-grid-view-migration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3746
+version: 1020.21.0
+---
+Firmware list to grid view migration.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-21-0-firmware-list-to-grid-view-migration.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the user experience and provide a more intuitive overview, the firmware list has been migrated from a simple list view to a grid view. The grid view displays the firmware items in a structured manner with additional details such as the firmware name, version, and compatible devices at a glance. This change enhances the readability and navigation of the firmware list, making it easier for users to find and manage their firmware items within the application.
```

File: content/change-logs/device-management/ui-c8y-1020-21-0-firmware-list-to-grid-view-migration.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-21-0-firmware-list-to-grid-view-migration.md:17]:
```suggestion
To improve the user experience and to ensure consistency, the firmware list has been migrated from a simple list view to a grid view.
```

**Review by dafkk — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@aaronraab Please review and approve

**Pull request merged**